src/script/pages/app-publish.ts

```diff
@@ -102,6 +120,45 @@ export class AppPublish extends LitElement {
     `;
   }
 
+  async generatePackage(type: platform) {
+    console.log('generating package', type);
+    switch (type) {
+      case 'windows':
+        try {
+          // eslint-disable-next-line no-case-declarations
+          const options = createWindowsPackageOptionsFromManifest('anaheim');
+
+          await generateWindowsPackage('anaheim', options);
```

Comment:
Just a thought, would it make sense to put the flavors in a type in the windows-publish file?
I don't think it'll affect much, but might be nice with we are bundling it out in the future, like as a separate package.